tests/test_buildah_baseline.sh

```diff
@@ -74,9 +74,9 @@ scratchmnt=$(buildah mount $newcontainer)
 echo $scratchmnt
 
 ########
-# Install Fedora 26 bash and coreutils
+# Install Fedora 27 bash and coreutils
```

Comment:
I'm not sure how these two *.sh files got into this commit?  They were part of #349 which I did this weekend and was merged yesterday by @rhatdan?